lib/utils/timezones.ts

```diff
@@ -1198,5 +1198,5 @@ export const getTimeByTimezone = (offset: number) => {
   // using supplied offset
   const time = new Date(utc + 3600000 * offset);
 
-  return `${time.getHours()}:${time.getMinutes()}${time.getHours() > 11 ? "pm" : "am"}`;
+  return `${time.getHours()}:${`0${time.getMinutes()}`.slice(-2)}${time.getHours() > 11 ? "pm" : "am"}`;
```

Comment:
Let's conditionally prefix the minutes instead of slicing after:

```suggestion
  return `${time.getHours()}:${`${time.getMinutes() < 10  ? "0" : ""}${time.getMinutes()}}${time.getHours() > 11 ? "pm" : "am"}`;
```

Reply:
How about using the format method form "date-fns" library? It is already installed as a dependency.

```
export const getTimeByTimezone = (offset: number) => {
  var d = new Date();

  // convert to msec
  // subtract local time zone offset
  // get UTC time in msec
  const utc = d.getTime() + d.getTimezoneOffset() * 60000;

  // create new Date object for different city
  // using supplied offset
  const time = new Date(utc + 3600000 * offset);

  return format(time, "hh:mmaaa");
};

```

Reply:
Yes, using date-fns would be more predictable for formatting